hydroflow_lang/src/graph/ops/fold.rs

```diff
@@ -110,7 +110,7 @@ pub const FOLD: OperatorConstraints = OperatorConstraints {
                     };
                 },
                 quote_spanned! {op_span=>
-                    #context.schedule_subgraph(#context.current_subgraph(), false);
+                    // #context.schedule_subgraph(#context.current_subgraph(), false);
```

Comment:
hmm why is this here

Reply:
I think we added this to make fold stop spinning but this is no longer necessary.